conf/config.yaml

```diff
@@ -66,252 +213,230 @@ appservice:
     # Whether or not to receive ephemeral events via appservice transactions.
     # Requires MSC2409 support (i.e. Synapse 1.22+).
     ephemeral_events: true
-
     # Should incoming events be handled asynchronously?
     # This may be necessary for large public instances with lots of messages going through.
     # However, messages will not be guaranteed to be bridged in the same order they were sent in.
+    # This value doesn't affect the registration file.
     async_transactions: false
 
     # Authentication tokens for AS <-> HS communication. Autogenerated; do not modify.
     as_token: "This value is generated when generating the registration"
     hs_token: "This value is generated when generating the registration"
 
-# Prometheus config.
-metrics:
-    # Enable prometheus metrics?
-    enabled: false
-    # IP and port where the metrics listener should be. The path is always /metrics
-    listen: 127.0.0.1:8000
-
-signal:
-    # Default device name that shows up in the Signal app.
-    device_name: mautrix-signal
+    # Localpart template of MXIDs for remote users.
+    # {{.}} is replaced with the internal ID of the user.
+    username_template: signal_{{.}}
```

Comment:
That part had a huge unintended side effect for me, it renamed all the puppet users. They now have no display name nor avatar. What do you think we should do. Make it configurable? Add more tests during migration? Revert that change? This may cause more breakage for users who have installed it in between.